### ISSUE TITLE
Exclamation point warning on all caps text

### ISSUE THREAD

**Issue by CatherineH** [amperser/proselint#540]
I am using proselint 0.6.0 with python 3.5. I have the following sample text:

```
Hello, yes I AM A DOG.
proselint rules!!!!!
```

This produces the proselint warning:

```
$ proselint proselint_test.txt 
proselint_test.txt:1:11: leonard.exclamation.multiple Stop yelling. Keep your exclamation points under control.
proselint_test.txt:2:11: hyperbolic.misc 'rules!!!!!' is hyperbolic.
```
It seems like leonard.exclamation.multiple is catching multiple capitalized words rather than exclamation points, so it is confusing that it warns me about the latter.

**Comment by joshmgrant** [comment]
I'm able to reproduce this in 0.6.1 with Python 2.7. It looks like a bug, either an incorrect error message or incorrect logic in the check.

**Comment by suchow** [comment]
Here's the relevant code:

```

@memoize
def check_repeated_exclamations(text):
    """Check the text."""
    err = "leonard.exclamation.multiple"
    msg = u"Stop yelling. Keep your exclamation points under control."

    regex = r"[^A-Z]\b((\s[A-Z]+){3,})"

    return existence_check(
        text, [regex], err, msg, require_padding=False, ignore_case=False,
        max_errors=1, dotall=True)
```

You're right, this is incorrect — it's checking for > 2 sequential all-caps words, not > 1 sequential exclamation points.

Good catch.

**Comment by suchow** [comment]
(The fix is to alter the regex to match 2+ sequential exclamation points.)

**Comment by michaelpacer** [comment]
Thank you @CatherineH for letting us know! 

Would you pull @Decagon's fix and test it to make sure it now works appropriately on your build? 

Also, @CatherineH, would you be willing to write a test for this?
